rfc/rfc005-distributed-network-using-grpc.md

```diff
@@ -206,18 +201,14 @@ A peer that floods the node with many messages threatens the stability of a node
 for processing incoming messages often have hard limits in the form connection pools, thread pools or backlogs.
 
 Countermeasures:
-- Nodes SHOULD limit the number of messages received from a peer.
-
-TODO: What would be a good number? 5/second?
+- Nodes SHOULD limit the number of network messages received from a peer to 5 per second.
```

Comment:
Maybe limit sending to 5/sec. You can't do a lot about receiving too much other than signaling it.